mdoc/Test/DocTest-typeForwards.cs

```diff
@@ -3,6 +3,11 @@
 [assembly:TypeForwardedToAttribute(typeof(TheNamespace.TheClass))] 
 #endif
 
+#if THIRD
```

Comment:
Not a huge deal, but strictly speaking this is the same as the `TWO` ... so you could simplify this declaration by doing

```
#if SECOND || THIRD
```

Reply:
Thanks for the suggestion, improved